src/client/app/components/DashboardComponent.jsx

```diff
@@ -5,16 +5,17 @@
 import React from 'react';
 import UIOptionsContainer from '../containers/UIOptionsContainer';
 import LineChartContainer from '../containers/LineChartContainer';
+import BarChartContainer from '../containers/BarChartContainer';
 
 /**
  * React component that controls the dashboard
  */
-export default function DashboardComponent() {
+export default function DashboardComponent(props) {
 	return (
 		<div className="container-fluid">
 			<UIOptionsContainer />
 			<div className="col-xs-10">
-				<LineChartContainer />
+				{props.chartToRender === 'line' ? <LineChartContainer /> : <BarChartContainer />}
```

Comment:
`'line'` really feels like it should be defined as an enum-like variable somewhere instead of just a string literal.